patches/check-links+2.1.2.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/node_modules/check-links/lib/check-link.js b/node_modules/check-links/lib/check-link.js
```

Comment:
patched as it's a bit chatty and logs the full response json out when there is a failure.  Quickly becomes too much